guides/common/modules/downloading_installing_satellite_disconnected.adoc

```diff
@@ -131,4 +131,4 @@ baseurl=file:///media/rhel7-server/
 # ./install_packages
 ----
 +
-If you have successfully installed the Satellite packages, the following message is displayed: `Install is complete. Please run satellite-installer --scenario satellite`.
+If you have successfully installed the {Project} packages, the following message is displayed: `Install is complete. Please run {foreman-installer} --scenario satellite`.
```

Comment:
I assume you're already aware of the "satellite" in this command.

Reply:
yep. This is something that we'll have to tackle individually as is very different in Foreman